pkg/config/remote/service/service.go

```diff
@@ -326,7 +331,7 @@ func (s *Service) Start(ctx context.Context) {
 	}()
 }
 
-//nolint:revive // TODO(RC) Fix revive linter
+// Stop stops the refresh loop and closes the on-disk DB cache
```

Comment:
```suggestion
// Stops the refresh loop and closes the on-disk DB cache
```

Reply:
I would prefer this but the Go mantra is "<Optional Article> <Name of struct|func> ....." for comments. So this would upset the linter.